src/server_manager/install_scripts/install_server.sh

```diff
@@ -118,6 +136,9 @@ function verify_docker_running() {
   fi
 }
 
+function install_curl() {
+  apt update && apt upgrade && apt install curl | sh > /dev/null 2>&1
```

Comment:
Do we need `apt upgrade`?

Reply:
`apt update` gets the version info. `apt upgrade` will attempt to upgrade all of the installed packages. Definitely not needed to get the latest version of curl.